scripts/check-vsix-size.ts

```diff
@@ -12,7 +12,7 @@ const vsixFileName = path.resolve(
 );
 const size = fs.statSync(vsixFileName).size;
 
-const maxSize = 7 * 1000000; // 7 MB
+const maxSize = 8 * 1000000; // 8 MB
```

Comment:
This isn't a great thing to increase, going to investigate in a follow up what's driving it up. (This pr increases it ~1mb)